src/components/HomePage.jsx

```diff
@@ -25,7 +27,7 @@ const HomePage = () => {
 						{articlesList.map((article) => {
 							return (
 								<div className="col" key={article.article_id}>
-									<div className="card" style={{ width: '18rem' }}>
+									<div className="card" style={{ width: '25rem' }}>
```

Comment:
It's always good to try and use a more semantic html than a div. It's also best to keep our styling extracted into our css files to be able to work on it easier and keep things organised.